internal/zoneinfo/zoneinfo_clock_test.go

```diff
@@ -0,0 +1,207 @@
+package zoneinfo_test
+
+import (
+	"testing"
+	"time"
+
+	"github.com/influxdata/flux/internal/zoneinfo"
+)
+
+func TestLocation_ToLocalClock(t *testing.T) {
+	const (
+		US_Pacific     = "America/Los_Angeles"
+		Australia_East = "Australia/Sydney"
+		American_Samoa = "Pacific/Apia"
+	)
+	for _, tt := range []struct {
+		name    string
+		locname string
+		s       string
+		want    string
+	}{
+		{
+			name:    "US_Pacific",
+			locname: US_Pacific,
+			s:       "2017-02-24T12:00:00Z",
+			want:    "2017-02-24T12:00:00-08:00",
+		},
+		{
+			name:    "US_Pacific DST",
+			locname: US_Pacific,
+			s:       "2017-09-03T12:00:00Z",
+			want:    "2017-09-03T12:00:00-07:00",
+		},
+		{
+			name:    "US_Pacific DST start",
+			locname: US_Pacific,
+			s:       "2017-03-12T02:30:00Z",
+			want:    "2017-03-12T03:00:00-07:00",
+		},
+		{
+			name:    "US_Pacific DST end",
+			locname: US_Pacific,
+			s:       "2017-11-05T01:30:00Z",
+			want:    "2017-11-05T01:30:00-07:00",
+		},
+		{
+			name:    "Australia_East",
+			locname: Australia_East,
+			s:       "2017-09-17T12:00:00Z",
+			want:    "2017-09-17T12:00:00+10:00",
+		},
+		{
+			name:    "Australia_East DST",
+			locname: Australia_East,
+			s:       "2017-02-24T12:00:00Z",
+			want:    "2017-02-24T12:00:00+11:00",
+		},
+		{
+			name:    "Australia_East DST start",
+			locname: Australia_East,
+			s:       "2017-10-01T02:30:00Z",
+			want:    "2017-10-01T03:00:00+11:00",
+		},
+		{
+			name:    "Australia_East DST end",
+			locname: Australia_East,
+			s:       "2017-04-02T02:30:00Z",
+			want:    "2017-04-02T02:30:00+11:00",
+		},
+		{
+			name:    "American_Samoa day skip morning",
```

Comment:
Should we include similar test cases for American Samoa in the `FromLocalClock` tests? Maybe there's a good reason why we didn't, but that reason is not obvious to me.

Reply:
It's not really possible unfortunately. Because the time doesn't exist, it's not really clear what we would be testing. On the other hand, going from UTC to the local timezone has to verify the behavior of "what if this time doesn't exist?"